src/appkit.cs

```diff
@@ -27918,11 +27929,14 @@ interface NSSearchToolbarItem
 		[Export ("resignsFirstResponderWithCancel")]
 		bool ResignsFirstResponderWithCancel { get; set; }
 
-		[Export ("minimumWidthForSearchFieldRepresentation")]
-		nfloat MinimumWidthForSearchFieldRepresentation { get; set; }
+		[Export ("preferredWidthForSearchField")]
```

Comment:
Availability (here and below) if this is newer than `[Mac (10,9)]`?

I'm also a little confused re the removal of `minimumWidthForSearchFieldRepresentation` and `preferredWidthForSearchFieldRepresentation` - usually if we remove APIs it's a breaking change since customers may be using the API. Were the names of these APIs changed/removed between betas for Xcode 12? 

It's also likely that I'm missing something here so please feel free to correct me @tj-devel709 :)

Reply:
Yeah I think you're totally right about missing the `[Mac (11,0)]` for these, thanks! 
And yes the `MinimumWidthForSearchFieldRepresentation` and `PreferredWidthForSearchFieldRepresentation` were introduced in xcode12 beta1!